test/small1/decl1.c

```diff
@@ -1,20 +0,0 @@
-struct timeval {
-	int	tv_sec;		 
-	int	tv_usec;	 
-};
-
-extern struct timeval xtime;
-
-
-volatile struct timeval xtime __attribute__ ((aligned (16)));
```

Comment:
The two declared types of `xtime` are not compatible due to the difference in the `volatile` qualifier. Checked in Godbolt that both GCC and Clang consider this a compilation error, so there's no need for CIL to accept this either.